sycl/test/check_device_code/no_offset.cpp

```diff
@@ -0,0 +1,46 @@
+// RUN: %clangxx -fsycl-device-only -fsycl-early-optimizations -fsycl-dead-args-optimization -D__SYCL_DISABLE_PARALLEL_FOR_RANGE_ROUNDING__ -S -emit-llvm -o - %s | FileCheck %s
```

Comment:
```suggestion
// RUN: %clangxx -fsycl-device-only -emit-llvm -o - %s | FileCheck %s
```
do we need these options?

Reply:
Yes, we need these options here, because that feature have to decrease amount of arguments. If we don't use these flags, optimization will not be turned on.